internal/dnsdocs/generator.go

```diff
@@ -0,0 +1,131 @@
+package main
```

Comment:
Maybe worth attaching a `//go:generate` comment here to allow `go generate ./...` to trigger this as well?